Alert/README.md

```diff
@@ -0,0 +1,32 @@
+# Alert data model
+
+## Description
+This entity models an alert generated either by a user or by a device. This models could be used to send alerts related to traffic jam, accidents, weather conditions, high level of pollutants and so on.
+The purpose of the model is to support the generation of notifications for a user or trigger other actions,
+based on such alerts.
+
+
+## Alert definition
+Alerts are context data generated by final users (humans) or devices, such as smartphones, service, or sensors. 
+
+An alert is generated by a specific situation. The main features of an alert is that it is not predictable and it is not a recurrent data. That means that an alert could be an accident or a high level of pollutants measure, additionally it could be the fall down of a patient or a car driving in the opposite direction.
+
+When a user or device detects an irregular situation sends an alert to a set of users. The context data around the alert are send to the Orion Context Broker according a specific data model. Some examples of context data are: type of alert (traffic, suspicious activities, and pollution, etc.), severity, location and so on.
```

Comment:
here I would not talk about Orion Context Broker, the data model should be independent of particular implementations

Reply:
I would not say anything about who receives the alert or how the alert is consumed.